kitodo-mediaserver-core/src/main/resources/config/default.yml

```diff
@@ -15,6 +15,15 @@ spring:
     password: kitodo
   main:
     banner-mode: "off"
+  mail:
+    host: localhost
+    username:
+    password:
+    smtp:
```

Comment:
Is this key used anywhere? (it should)
Also there should be a possibility to use TLS implicitly and to set a port number.

Reply:
It's used automatically by Spring for JavaMail. You could use port 465 to get implicit TLS. And therefore, yes, you are right, port setting is missing.